### PR TITLE
Code Cleaning

### PR DESCRIPTION
**Pull request by Borda**
minor code cleaning
consider adding code quality check which may improve the code style, e.g. codacy
... [Codacy Badge](https://www.codacy.com/app/Borda/dipy?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Borda/dipy&amp;utm_campaign=Badge_Grade)

### CODE REVIEW

**Comment by Borda** [comment]
it seems that the codecov/path is not much relevant since the corrected lines have not sometimes touch bu previous tests and now it marks as own issue...

**Comment by Borda** [comment]
@skoudoro @arokem ^^

File: README.rst [hunk 1]
```diff
@@ -12,6 +12,9 @@
 .. image:: https://codecov.io/gh/nipy/dipy/branch/master/graph/badge.svg
   :target: https://codecov.io/gh/nipy/dipy
 
+.. image:: https://api.codacy.com/project/badge/Grade/65d6b1c8dfb74f2d8401cd213f55d710
+  :target: https://www.codacy.com/app/Borda/dipy?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Borda/dipy&amp;utm_campaign=Badge_Grade
+
```
Comment (skoudoro) [inline README.rst]: This badge can be created later, we need to create a specific account for DIPY. So you can remove it for the moment. 
What do you think @arokem?

**Comment by Borda** [comment]
regarding removing variables, since they are commented it would take more time to find them and remove them, on the other hand, there is more commented code even before my changes so I would rather propose to address it in another PR - removing commented code...

**Comment by skoudoro** [comment]
Hi @Borda, It is much easier to remove them with a diff on this PR than tracking and removing them after a merge in a new PR. I do not see your point here... Can you explain to me why it would take more time to do it on this PR than another one?

**Comment by Borda** [comment]
OK, I remove it, but there is still other commented code...

**Pull request merged**